app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  belongs_to :branch
+  has_many :accounts, dependent: :destroy
+  has_many :loans
+  validates :name,:father_name,:mother_name,:age,:address,:contact_no,presence: true
```

Comment:
add validation on age

Reply:
But we can create an account less than age 18..

Reply:
yes but can the age be less then 0
i can enter -1 age in it

Reply:
Got it